docs/source/guide/cdr-1-intro.myst

```diff
@@ -0,0 +1,137 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# How do I use CDR?
+
+Here we show how to use Clifford data regression (CDR) by means of a simple example. CDR is a learning-based quantum error mitigation technique that relies on the training of a classical simulator with quantum circuits that resemble the one that is the aim of the quantum error mitigation, but which are easier to simulate {cite}`Czarnik_2021_Quantum`. More information about this technique can be found in the section [What is the theory behind CDR?](cdr-5-theory.myst).
+
+```{code-cell} ipython3
+import warnings
+warnings.filterwarnings("ignore")
+
+import numpy as np
+
+import cirq
+from mitiq import cdr, Observable, PauliString
+```
+
+## Problem setup
+
+To use CDR, we call {func}`.cdr.execute_with_cdr` with four "ingredients":
+
+1. A quantum circuit to prepare a state $\rho$.
+1. A quantum computer or noisy simulator to return a {class}`.QuantumResult` from $\rho$.
+1. An observable $O$ which specifies what we wish to compute via $\text{Tr} [ \rho O ]$.
+1. A near-Clifford (classical) circuit simulator.
+
++++
+
+### 1. Define a quantum circuit
+
+The quantum circuit can be specified as any quantum circuit supported by Mitiq but **must be compiled into the gateset $\{ \sqrt{X}, Z, \text{CNOT}\}$.**
+
+```{code-cell} ipython3
+a, b = cirq.LineQubit.range(2)
+circuit = cirq.Circuit(
+    cirq.rx(0.1).on(a),
+    cirq.rx(-0.72).on(b),
+    cirq.rz(0.4).on(a),
+    cirq.rz(0.2).on(b),
+    cirq.CNOT.on(a, b),
+    cirq.rx(-0.1).on(b),
+    cirq.rz(-0.23).on(a),
+    cirq.CNOT.on(b, a),
+    cirq.rx(-0.112).on(a),
+)
+circuit
+```
+
+### 2. Define an executor
+The executor inputs a circuit and returns a {class}`.QuantumResult`. Typically this function will send the circuit to a quantum computer and wait for the results. Here for sake of example we use a simulator that adds single-qubit depolarizing noise after each moment and returns the final density matrix.
```

Comment:
```suggestion
We define an [executor](guide-executors.myst) function which inputs a circuit and returns a {class}`.QuantumResult`. Here for sake of example we use a simulator that adds single-qubit depolarizing noise after each moment and returns the final density matrix.
```
Simplify with a link to Executors section and avoid mentioning hardware to avoid confusion with density matrix simulation.